couchdb-cluster-admin/copy_db_to_new_cluster.py

```diff
@@ -64,18 +64,24 @@ def _copy_db_doc(from_details, to_details, db_name):
     parser.add_argument('--to-username', dest='to_username',
                         help='Admin username for "to" cluster (defaults to the same as "from-username"')
     parser.add_argument('--database', dest='database', help='Database to copy or "ALL"')
+    parser.add_argument('--socks', dest='socks', help='Use a socks proxy at given port for '
+                                                      'communication with the "from" cluster')
```

Comment:
Might be helpful to add a note that the value to pass in is the port (assuming I read things correctly)?

Reply:
will rephrase to make clearer